pkg/doc/did/doc.go

```diff
@@ -677,21 +698,33 @@ func populateRawPublicKey(context string, pk *PublicKey) map[string]interface{}
 		rawPK[jsonldController] = pk.Controller
 	}
 
-	if pk.Value != nil {
+	if pk.jsonWebKey != nil {
```

Comment:
The jsonWebKey is not exposed. How can the user set this ?

Reply:
Should we give a user ability to set it? It's got defined when being unmarshalled from bytes.

Reply:
How would this work with current version ? How does `pk.jsonWebKey` get set ?

Reply:
Reverted back to exposed `JSONWebKey`.